webviz_ert/controllers/response_correlation_controller.py

```diff
@@ -241,8 +241,7 @@ def update_response_overview_plot(
 
         fig.update_layout(_layout_figure(x_axis_label))
 
-        default_index = 0
-        x_index = corr_xindex.get(selected_response, default_index)
+        x_index = corr_xindex.get(selected_response)
```

Comment:
This `default_index` was added to be explicit about the default selection of this `get` function. It makes it easier to understand that if corr_xindex was empty for a given `selected_response`, `x_index` would be zero. The proposal of this commit is DRYer but let implicit this choice. Do you see a way to be explicit and also DRY?

Reply:
I do appreciate being explicit, but I'm not sure if I want to be explicit in that spot - if i was reading the code, and wanted to know how that `corr_xindex` gets calculated / set, i would try to find the function manipulating it (in this case, the callback that has it as output), and check there. and indeed, in that callback, `_get_default_x_index` is called, which has as last line `return 0`.

If you feel like we want to be explicit in the plot callback as well, no problem, I can create a variable scoped to the controller, which holds the default index value `0`, and use it in both places (in the `_get_default_x_index` function, and in the response overview plot callback). Then it's easier to see the default value in that context, and it's DRY, so no problem with updating the value.

what do you think?

Reply:
I think that is actually a very good suggestion! In this way we are explicit and dry!

Reply:
done : )